--- autounit-0.15.2/doc/Makefile.am.orig	2002-04-23 00:50:08.000000000 +0200
+++ autounit-0.15.2/doc/Makefile.am	2003-11-09 18:36:42.772239904 +0100
@@ -10,18 +10,3 @@
 			au-concept-index.texi
 
 EXTRA_DIST = $(autounit_TEXINFOS)
-docdir = $(prefix)/doc/$(PACKAGE)-$(VERSION)
-doc_DATA = autounit.html
-
-
-SUFFIXES = .html
-html_docs = autounit.html
-autounit.html: autounit.info
-.texi.html:
-	$(MAKEINFO) -I $(top_srcdir)/doc --html $<
-
-.PHONY: html
-html: version.texi $(html_docs)
-
-clean-local:
-	if test -d autounit ; then rm -rf autounit ; fi
